public/docs/ts/latest/guide/appmodule.jade

```diff
@@ -0,0 +1,136 @@
+include ../_util-fns
+
+:marked
+  An Angular Module class describes how the application parts fit together.

```

Comment:
@chalin Agreed. I have been making them lower case and linking sometimes (if it seems appropriate in the context). In this case, yes, I agree that linking would be good, too.